beets/ui/__init__.py

```diff
@@ -1133,9 +1136,13 @@ def _configure(options):
     else:
         log.set_global_level(logging.INFO)
 
+    if overlay_path:
+        log.debug(u'overlaying configuration: {0}',
+                  util.displayable_path(config_path))
+
     config_path = config.user_config_path()
     if os.path.isfile(config_path):
-        log.debug(u'user configuration: {0}',
+        log.debug(u'base user configuration: {0}',
```

Comment:
This might make the output a tad more confusing when there is no overlay. Can we please leave it as just "user configuration"?